core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/GroupByAggregator.java

```diff
@@ -135,6 +135,10 @@ public class GroupByAggregator extends RecordReducibleAggregator<AggregateResult
       put(GroupByConfig.Function.CONCATDISTINCT, "STRING_AGG(DISTINCT CAST(%s AS STRING) , \", \")");
       put(GroupByConfig.Function.LOGICALAND, "COALESCE(LOGICAL_AND(%s), TRUE)");
       put(GroupByConfig.Function.LOGICALOR, "COALESCE(LOGICAL_OR(%s), FALSE)");
+      put(GroupByConfig.Function.SHORTESTSTRING,
+          "STRING_AGG(CAST(%s AS STRING) ORDER BY LENGTH(CAST(%<s AS STRING)) ASC LIMIT 1)");
+      put(GroupByConfig.Function.LONGESTSTRING,
+          "COALESCE(STRING_AGG(CAST(%s AS STRING) ORDER BY LENGTH(CAST(%<s AS STRING)) DESC LIMIT 1), \"\")");
```

Comment:
Why do you need coalesce just in one of the functions? According to docs `string_agg` skips nulls.

Reply:
Because the Shortest String aggregator returns `null` for no input values, but the Longest String aggregator returns `""` for the same input.

Reply:
Fixed https://cdap.atlassian.net/browse/PLUGIN-1210 alongside this PR